__tests__/app.test.js

```diff
@@ -137,3 +137,59 @@ describe("GET /api/articles", () => {
       });
   });
 });
+
+describe("GET /api/articles/:article_id/comments", () => {
+  test("200- should return an array of comment objects for a given article id, with each comment object containing the correct properties", () => {
+    return request(app)
+      .get("/api/articles/3/comments")
+      .expect(200)
+      .then(({ body: { comments } }) => {
+        expect(comments).toEqual([
+          {
+            comment_id: 11,
+            body: "Ambidextrous marsupial",
+            article_id: 3,
+            author: "icellusedkars",
+            votes: 0,
+            created_at: "2020-09-19T23:10:00.000Z",
+          },
+          {
+            comment_id: 10,
+            body: "git push origin master",
+            article_id: 3,
+            author: "icellusedkars",
+            votes: 0,
+            created_at: "2020-06-20T07:24:00.000Z",
+          },
+        ]);
+      });
+  });
+  test("should return an array of comment objects sorted by creation date in descending order", () => {
+    return request(app)
+      .get("/api/articles/3/comments")
+      .then(({ body: { comments } }) => {
+        expect(comments).toBeSortedBy("created_at", {
+          descending: true,
+          coerce: true,
+        });
+      });
+  });
+   test("404- should return correct error message when article_id is valid, but it doesn't exist in the database", () => {
+    return request(app)
+      .get("/api/articles/88/comments")
+      .expect(404)
+      .then(({ body: { message } }) => {
+        expect(message).toBe("Not Found: Non-Existent Article ID");
+      });
+  });
+      test("400- should return correct error message when given invalid article_id", () => {
+        return request(app)
+          .get("/api/articles/bestarticle/comments")
+          .expect(400)
+          .then(({ body: { message } }) => {
+            expect(message).toBe("Bad Request: Invalid Article ID");
+          });
+      });
+  });
```

Comment:
Great error handling tests here 👍 
One more test is needed here:
200 - getting the comments of an article which has no comments should return no error and an empty array